packages/Articles/Controllers/CategoryController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace PhpSoft\Articles\Controllers;
+
+use Input;
+use Validator;
+use Illuminate\Http\Request;
+use App\Http\Requests;
+
+class CategoryController extends Controller
+{
+    private $categoryModel = '';
+
+    /**
+     * Construct controller
+     */
+    public function __construct()
+    {
+        $this->categoryModel = config('phpsoft.article.categoryModel');
+    }
+
+    /**
+     * Create resource action
+     *
+     * @param  Request $request
+     * @return Response
+     */
+    public function store(Request $request)
+    {
+        $validator = Validator::make($request->all(), [
+            'name'        => 'required',
+            'alias'       => 'regex:/^[a-z0-9\-]+/|unique:article_categories',
```

Comment:
Your code this and migration are not consistency about unique. But I think alias is not unique. It just for SEO or something. API handle model just by `id`.